python_modules/libraries/dagster-aws/dagster_aws/ecs/tasks.py

```diff
@@ -74,8 +74,7 @@ def default_ecs_task_definition(
     # and the command will fail
     # https://aws.amazon.com/blogs/opensource/demystifying-entrypoint-cmd-docker/
     container_definitions = task_definition["containerDefinitions"]
-    container_definitions.remove(metadata.container_definition)
-    container_definitions.append(
+    container_definitions = [
```

Comment:
The thing I'm a bit hesitant about here is if anybody is relying on this to provide sidecar containers in their job run task. But I think if somebody wants that behavior, they should provide their own custom task definition via the launcher's config.

Reply:
(Which you can't do yet in cloud sadly, but we should fix that)